src/test/java/jalau/at18/katas/phonenumber/daniela/PhonenumberTest.java

```diff
@@ -0,0 +1,14 @@
+package jalau.at18.katas.phonenumber.daniela;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class PhonenumberTest {
+    @Test
+    public void testPhonenumber() {
+        Phone phone = new Phone();
+        final Integer[] inputArray1 = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
+        var outnumber = "(123)456-7890";
```

Comment:
Missing space between ")" and "4". Please update it so it matches kata requirement
```
createPhoneNumber([1, 2, 3, 4, 5, 6, 7, 8, 9, 0]) // => returns "(123) 456-7890"
```